core/src/main/java/org/infinispan/statetransfer/StateTransferManagerImpl.java

```diff
@@ -120,7 +130,9 @@ public void rebalance(CacheTopology cacheTopology) {
       }, partitionHandlingManager);
 
       if (trace) {
-         log.tracef("StateTransferManager of cache %s on node %s received initial topology %s", cacheName, rpcManager.getAddress(), initialTopology);
+         if (initialTopology != null) {
```

Comment:
I was trying to avoid a potentially confusing message in the logs, but I guess "initial topology null" is unambiguous